src/briefcase/integrations/linuxdeploy.py

```diff
@@ -119,3 +119,71 @@ def patch_elf_header(self):
                 # The original or patched value. If this is the case, the file is likely
                 # wrong and we should raise an exception.
                 raise CorruptToolError("linuxdeploy")
+
+
+class LinuxDeployGtkPlugin:
```

Comment:
Is there any potential for a *generic* plugin interface here? I noticed there is a Qt plugin - it seems likely we may need to use that for PySide2/6 deployments

Reply:
I think there definitely is a potential for something more general in the future if we add additional plugins. :+1: